examples/nodejs-local/typescript/src/main.ts

```diff
@@ -1,9 +1,7 @@
 import { DevCycleClient, initializeDevCycle } from '@devcycle/nodejs-server-sdk'
 import { DVCClientAPIUser } from '@devcycle/types'
 import { plainToInstance } from 'class-transformer'
-import { Query } from 'express-serve-static-core'
```

Comment:
this is a good example of why we should use pnpm

this code was only working because yarn was hoisting this dependency to the root of node_modules so it was able to be found, even though it wasn't directly declared as a dependency anywhere in our repo.